admin-portal/src/config.json

```diff
@@ -1,8 +1,8 @@
 {
   "clientID": "W_ZaTLGR20OKy0HTYPqSAJvuS5Ia",
   "baseUrl": "https://api.asgardeo.io/t/elixir",
-  "signInRedirectURL": "https://admin-portal-rouge.vercel.app",
-  "signOutRedirectURL": "https://admin-portal-rouge.vercel.app",
+  "signInRedirectURL": "https://localhost:3000",
```

Comment:
We don't need to commit this right?

Reply:
yes

Reply:
Reverted this file to the previous commit. Thank you.